src/lib/iptables.ts

```diff
@@ -98,10 +102,11 @@ export function convertToRestoreRulesFormat(rules: Rule[]): string {
 				if (rule.matches) {
 					rule.matches.forEach((match) => args.push(match));
 				}
-				if (rule.comment) {
-					args.push('-m comment');
-					args.push(`--comment "${rule.comment}"`);
-				}
+				// TODO: Once comments are universally supported in meta-balena we can enable this
+				// if (rule.comment) {
+				// 	args.push('-m comment');
+				// 	args.push(`--comment "${rule.comment}"`);
+				// }
```

Comment:
Is there any way to feature detect this?

Reply:
@Page- we want to but we want to think through the best way to do the detection in more "general" way; perhaps contracts etc